docs/CloudPermissions.md

```diff
@@ -0,0 +1,58 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
+This guide explains how to assign required permissions for the users for deploying Scalar DL on Azure.
```

Comment:
Is this guide for k8s? or for scalar dl?
If you say scalar dl, then it will mean, users would need the permissions when they deploy scalar dl with terraform.
Is it correct?

Reply:
The permissions are for creating and managing all the resources that are needed for Scalar DL deployment. 
Specified in Azure section that the command line tools also can be used with these permissions.